pages/Messages/Overview.md

```diff
@@ -58,13 +58,13 @@ We have seriously considered two possibilities, a [variable announcement format]
 
 ### DSNP Announcement
 
-These announcements would be added to batch files for announcement on the blockchain.
+These announcements would be added to batch files for announcement on the blockchain. Some fields on the announcement are used as bloom filters for querying data in a batch. See [Batches Overview](/Batches/Overview#Queries)
```

Comment:
Can you separate each sentence to its own line? It doesn't affect presentation, but it makes for better diffs down the road.